Newsletters: add 245 (2023-04-05)

harding opened the pull request:
- [x] Add lede, update releases/RCs, add topic links @harding 
- [x] Fill-in new topic pages for Proof of Payment and Wallet Labels @harding
- [x] Core Lightning 5967 @bitschmidty 
- [x] LDK 2125 @glozow 
- [x] BTCPay Server 4826 @Xekyo 

(Sorry there were no Bitcoin Core PRs; if the alternative PRs are too alien to you, let me know and I'll write them up.)

paulkania commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,141 @@
+  regular forwarding information and are provided to the spender or
+  sender who uses them to send a payment to the first of the hops.  That
+  hop begins the process of decrypting the next hop, forwarding the
+  payment to it, having that hop decrypted the subsequent hop, etc,
```
'having that hop decrypted the subsequent hop'  decrypted -> decrypt

LarryRuane commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,142 @@
+    solution to both problems.  Accumulators allow compactly proving a
+    particular element is a member of a large set of elements and also
+    make it efficient to add new elements to the set.  Some accumulators
+    also make it efficient to delete elements from the set.  In a
```
As-is isn't bad, but I see what you mean, maybe something like
> Accumulators allow compactly proving a particular element is a member of a large set of elements and allow new elements to be easily added.

I know that in general, "easily" doesn't have exactly the same meaning as "more efficient", but in this case, I think they will have the same meaning.

The next sentence, "Some accumulators ...", could be left as is.

LarryRuane commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,143 @@
+  For example, Alice wants to split a 900 sat total into two parts, but
+  both of the paths she chooses require 500 sat minimum amounts. With
+  this specification change, she can now send two 500 sat payments,
+  choosing to overpay by a total of 100 sats in order to use her
```
```suggestion
  choosing to overpay by a total of 100 sats to use her
```

harding replied:
Leaving this as-is; "in order to" just feels better to me here.  :shrug:

Xekyo commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,143 @@
+    public key and to use the corresponding private key to generate a
+    signature for any breach-detection data it accepted.  Alice could
+    then publish the data and the signature after an unresolved breach
+    to prove the watchtower failed its responsibility.  However, Segura
```
Sergi’s first family name is “Delgado”, “Segura” is his second family name. He also [refers to himself](https://twitter.com/sr_gi) as “Sergi Delgado” on Twitter.

```suggestion
    to prove the watchtower failed its responsibility.  However, Delgado
```

sr-gi replied:
It means safe (in feminine), but it is, indeed, also my second last name.

I usually go by Delgado because it makes things easier (which, while we are at it, means thin in Spanish lol)

Xekyo commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,143 @@
+      Watchtowers may only want to store data for a limited time, e.g.
+      they may accept payment for a particular term.
+
+    Segura suggests cryptographic accumulators provide a practical
```
```suggestion
    Delgado suggests cryptographic accumulators provide a practical
```

Xekyo commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,143 @@
+    particular element is a member of a large set of elements and also
+    make it efficient to add new elements to the set.  Some accumulators
+    also make it efficient to delete elements from the set.  In a
+    [gist][segura watchtowers gist], Segura outlines several different
```
```suggestion
    [gist][segura watchtowers gist], Delgado outlines several different
```

bitschmidty reviewed (commented):
So far so good, no comments

Xekyo commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -71,7 +71,9 @@ Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
-- [Core Lightning #5967][] add listclosedchannels FIXME:bitschmidty
+- [Core Lightning #5967][] adds a `listclosedchannels` RPC that provides data
+  about the node's closed channels, including the cause of channel being closed.
+  Information about old peers is also now retained.
```
Nit:

```suggestion
  Information about old peers is also retained now.
```

harding commented:
Made edits for all feedback, thanks @paulkania @jonatack @LarryRuane @Xekyo @sr-gi @bitschmidty !  Added lede, filled in topic pages, updated releases/RCs, and added topic links.  Reviewed contributions by @glozow @bitschmidty @Xekyo (thanks!).

Thanks everyone!

jirijakes commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -0,0 +1,151 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.16.0-beta][] is the for a new major version of this popular LN
```
```suggestion
- [LND v0.16.0-beta][] is beta release of a new major version of this popular LN
```

or something similar.

bitschmidty replied:
Thanks for the catch, @jirijakes !

sr-gi commented on _posts/en/newsletters/2023-04-05-newsletter.md:
```diff
@@ -129,17 +133,19 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[lnd v0.16.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [segura watchtowers post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-March/003892.html
 [segura watchtowers gist]: https://gist.github.com/sr-gi/f91f007fc8d871ea96ead9b27feec3d5
```
nit: given all other references have been updated from segura to delgado, these may also need updating.

bitschmidty commented:
Squashed and merged, thank you to @harding @glozow @Xekyo @LarryRuane @jonatack @sr-gi @paulkania and @jirijakes for contributing this week! 🚀
